google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -1255,6 +1260,447 @@ public static SignUrlOption withQueryParams(Map<String, String> queryParams) {
     }
   }
 
+  class V4PostPolicyOption implements Serializable {
```

Comment:
Can you split all V4PostPolicy helper classes into a separate file, except what's needed for `generateV4PresignedPostPolicy` ease of use such as `V4PostPolicyOption`?